server/lib/adapters/postgres.js

```diff
@@ -33,6 +34,19 @@ export function closeConnection(client) {
 }
 
 /**
+ * Validate Result specific for postgres database
+ * @param result
+ * @returns Array of errors
+ */
+
+export function validateResult(result) {
+  if (_.some(result.fields, (column) => column.dataTypeID === 114)) {
+    return ["Column from postgres database is in json format"];
```

Comment:
And maybe a more explicit error message saying that columns with json format are not allowed. 

dataTypeID `3802` `jsonb` are not allowed either